Riku/Riku/StateUpdate.cpp

```diff
@@ -0,0 +1,21 @@
+//Kacper Walasek
+#include "StateUpdate.h"
+#include "PlayerPatchHandler.h"
+void StateUpdate::handlePatch(std::shared_ptr<Patch> patch)
```

Comment:
Posiadanie kilku handlerów wiąże się z kilkukrotnym aplikowaniem każdego patcha.
W jakiej sytuacji chcielibyśmy tak robić?

Reply:
Idea była taka, że każdy handler ogarnia jakąś inną część patcha. Jeżeli jest patch który zmienia wiele rzeczy w stanie gry to jeden handler ogarnia zmiany zasobów graczy, inny zmiany budynków itd.

Reply:
a nie, ten handler tylko resourceChange zczytuje i tak. Jest ok, lepiej pójdę spać  :)